packages/shared-components/src/formio/menu-group/dateAndTimeGroup.ts

```diff
@@ -1,5 +1,4 @@
 import datePickerBuilder from '../components/core/datepicker/DatePicker.builder';
-import dayBuilder from '../components/core/day/Day.builder';
```

Comment:
Er det noen grunn til å beholde `Day.ts` og tilhørende builder og form?

Reply:
Måtte tenke meg om en gang til. Vi må jo støtte eksisterende skjemaer. Kanskje vi skulle laget en oppgave for å migrere + fjerne filer.